Source/WebCore/platform/audio/gstreamer/WebKitWebAudioSourceGStreamer.cpp

```diff
@@ -155,6 +145,25 @@ static GstAudioChannelPosition webKitWebAudioGStreamerChannelPosition(int channe
     return position;
 }
 
+static GstCaps* getGStreamerAudioCaps(float sampleRate, unsigned numberOfChannels)
+{
+    guint64 channel_mask = 0;
+    GstAudioChannelPosition* positions = g_new(GstAudioChannelPosition, numberOfChannels);
```

Comment:
`fastMalloc` is preferred in WebCore, but here maybe you could use a `Vector<GstAudioChannelPosition>` ?

Reply:
Sure, I've pushed a variant with `Vector<GstAudioChannelPosition>` now.